frontend/src/components/Dashboard/DashboardResultsTable.tsx

```diff
@@ -80,46 +98,66 @@ const DashboardResultsTable = (props: any) => {
                             </th>
                         )
                     )}
+                    <th className="border-2 border-blue-800 bg-blue-800 text-white" style={{ minWidth: "10rem" }}> Grading of session </th>
```

Comment:
Using an arbitrary value for min width is supported in Tailwind : 
```suggestion
                    <th className="border-2 border-blue-800 bg-blue-800 text-white min-w-[10rem]"> Grading of session </th>
```

Reply:
Done